src/trio/_timeouts.py

```diff
@@ -27,6 +27,9 @@ def move_on_after(seconds: float) -> trio.CancelScope:
     """Use as a context manager to create a cancel scope whose deadline is
     set to now + *seconds*.
```

Comment:
Maybe say "set to creation time + \*seconds\*"? Alternate phrasing to what you proposed.

Reply:
That would be cleaner. I was slightly worried about misinterpreting when the sentence starts with "Use as a context manager...", but I think that's a stretch.

Reply:
Should `fail_after` also get a similar treatment? Something like
> Creates a cancel scope with the given timeout, relative to creation time, and raises an error if it is actually cancelled.

or maybe we want one of them to be super explicit